### PR TITLE
Changes for ISAN

### PR DESCRIPTION
**Pull request by ignapas**
* Removed progress bars from studies in the Dashboard.
* Info buttons for hints now show at all times.
* Solved bug with long study titles in Dashboard.
* Settings hidden if node has no inputs
* Added service version in ServiceInfo view
* Added link to manual in NavBar
* Added error status for nodes that fail loading
* Help button inside user menu now links to new docs.
* Not showing dashboard for anonymous users.

### CODE REVIEW

File: services/web/client/source/class/qxapp/component/widget/NodesTree.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ qx.Class.define("qxapp.component.widget.NodesTree", {
       let data = {
         label: this.getStudyName(),
         children: this.__convertModel(topLevelNodes),
-        nodeId: "root",
+        nodeId: this.getWorkbench().getStudy().getUuid(),
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/component/widget/NodesTree.js]: Leave it as was, please

Comment (ignapas) [reply services/web/client/source/class/qxapp/component/widget/NodesTree.js]: It was a request from @pcrespov

Comment (ignapas) [reply services/web/client/source/class/qxapp/component/widget/NodesTree.js]: I just have to only show it when the user is tester

Comment (odeimaiz) [reply services/web/client/source/class/qxapp/component/widget/NodesTree.js]: I just discussed with him. Stays as "root"

Comment (ignapas) [reply services/web/client/source/class/qxapp/component/widget/NodesTree.js]: okay!

File: services/web/client/source/class/qxapp/desktop/NavigationBar.js [hunk 2]
```diff
@@ -87,6 +87,10 @@ qx.Class.define("qxapp.desktop.NavigationBar", {
       flex: 1
     });
 
+    this._add(new qxapp.ui.form.LinkButton(this.tr("Docs"), "https://docs.osparc.io").set({
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/desktop/NavigationBar.js]: Could the Help button also point to https://docs.osparc.io? (PO request)

**Pull request merged**